std_msgs/Header header

int32 id
float32 time
geometry_msgs/Vector3 tip_position
geometry_msgs/Point tilt
float32 pressure
